packages/dbos-cloud/cloudutils.ts

```diff
@@ -96,6 +96,11 @@ export function deleteCredentials() {
   fs.unlinkSync(`./${dbosEnvPath}/credentials`);
 }
 
+export function writeCredentials(credentials: DBOSCloudCredentials) {
+  execSync(`mkdir -p ${dbosEnvPath}`);
+  fs.writeFileSync(`${dbosEnvPath}/credentials`, JSON.stringify(credentials), "utf-8");
+}
```

Comment:
As suggested we should manage to change the umask of this file to 0600. I tried but I noticed even if we set it here, the resulting umask is 0644. This seem to be because the process umask takes precedence over what the code specifies. There's a [workaround](https://github.com/nodejs/node-v0.x-archive/issues/25756#issuecomment-125717353) but it might have other implications.